### DISCUSSION TITLE
New category for maritime heatwaves?

### DISCUSSION THREAD

**Discussion by fkromer** [protontypes/open-sustainable-technology#634] [Ideas]
Recently I‘ve heard about maritime heatwaves the first time. They are comparable with athmospheric heat waves but are way slower and last longer. Maritime heatwaves influence other environmental events like athmospheric heat waves. And their occurrence depends on a variety of characteristics (sun exposure -> temperature of water layers near the surface, wind -> heat exchange rate between different water layers, …). Consequently this topic does not fit perfectly well into the sections we have right now.

**Comment by Ly0n** [comment]
Hi @fkromer,
we have something like a heatwave section here: 
https://opensustain.tech/#ocean-carbon-and-temperature

If you are interested in this topic, you could help us find other projects in this area that meet the requirements. Maybe more from a broader perspective including Sea Surface Temperature (SST). So far I have only looked on GitHub for this topic, but maybe if you look hard you will find other relevant projects in this area on GitHub or other open source platforms.

**Reply by fkromer** [reply]
Ah, did not find the MHW projects right away. Sorry for that. So far I did not find new open source projects unfortunately.

According to https://www.marineheatwaves.org Australia seems to be leading in ML based research:

"FORECASTING MARINE HEATWAVES VIA MACHINE LEARNING - January 2021 - Australian team makes world-first marine heatwave predictions, with lead time of five months. The forecast was based on data to November 2020, and predicted a marine heatwave off the Western Australian coast beginning in January. ... Forecasts were based on a machine learning approach, and included observations on SST and ocean heat content."

Probably one could find out about open source projects via https://www.marineheatwaves.org/leads-and-contacts.html.

Over the longer run https://www.marineheatwaves.org/publications.html could be helpful as well (machine learning, data sets, ...).

**Reply by Ly0n** [reply]
> Probably one could find out about open source projects via [marineheatwaves.org/leads-and-contacts.html](https://www.marineheatwaves.org/leads-and-contacts.html).

It takes forever to contact people and ask them, because you are not going to get in touch with anyone who knows how to code via most contacts form on websites. 

The best way to do this is by picking the keywords from the text: SST, marine heatwave and ocean heat. When you search on GitHub always use the recently updated option. In this way you get the active project we are looking for. The best option is really not that helpful in most cases. In this way I just found this: https://github.com/robwschlegel/heatwave3

You are very welcome to create a PR and add this project.

**Reply by Ly0n** [reply]
I just looked in the people behind the code and the projects they made, stared and contributed to. In this way I found this: https://github.com/robwschlegel/MHWapp 

That the open source projects behind [marineheatwaves.org](https://www.marineheatwaves.org/) ^^

**Reply by fkromer** [reply]
Cool, thanks for this methodical blueprint for finding new projects 😄 Discussion can be closed I guess.
